front-end/src/Navbar.jsx

```diff
@@ -42,28 +43,40 @@ function Navbar({ username }) {
     }
 
     return (
-        <div className='navContainer'>
-            <div className='nav'>
-                <button className='profileIcon'>Profile</button>
-                <h1>SoundStitch</h1>
+        <Box w="70%" color="white" p={4} mx="auto">
+            <Flex align="center">
+                <button className="profileIcon">
+                    <Text fontSize={'lg'}>Profile</Text>
+                </button>
+                <Spacer />
+                <Heading as='h1' size='2xl' style={{ textAlign: 'center', flexGrow: 1 }}>SoundStitch</Heading>
+                <Spacer />
                 <div
                     className="menu"
                     onMouseEnter={handleMouseEnter}
                     onMouseLeave={handleMouseLeave}
                 >
-                    <button className='dropdownButton'>Menu</button>
+                    <button className="dropdownButton">
+                        <Text fontSize={'lg'}>Menu</Text>
+                    </button>
                     {isDropdownVisible && (
-                        <div className="dropdownMenu">
+                        <div className="dropdownMenu" style={{ position: 'absolute', right: 0 }}>
                             <ul>
-                                <li onClick={handleHome}>Home</li>
-                                <li onClick={handleCreate}>Create</li>
-                                <li onClick={handleLogout}>Logout</li>
+                                <li onClick={handleHome}>
```

Comment:
I'd probably advice more descriptive names here. `handleHome` doesn't tell me much, while `didClickHomeButton`, or `didClickHomeItem` makes more sense.

Reply:
Got it, will rename in the next commit.